packages/salesforcedx-vscode-core/src/commands/forceCreateManifest.ts

```diff
@@ -38,7 +39,9 @@ export class ManifestCreateExecutor extends LibraryCommandletExecutor<string> {
                    progress?: vscode.Progress<{ message?: string | undefined; increment?: number | undefined; }>,
                    token?: vscode.CancellationToken): Promise<boolean> {
     if (this.sourcePaths) {
+      const sourceApiVersion = (await SfdxProjectConfig.getValue('sourceApiVersion')) as string;
```

Comment:
What type is returned from get value here? Just wondering why you had to cast it to a string.

Reply:
When I ran it, it was telling me the type was string, but when I removed the cast, it gave me this error:
`Type 'string | number | boolean | JsonMap | JsonArray | null | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.`
So it seems like without the cast, the return type is undefined.

Reply:
word. That makes sense 👍